lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -458,7 +458,7 @@ def check_scope
           end
 
           def reload_position
-            self.reload
+            self.reload unless new_record?
```

Comment:
Yea I can't see the harm in removing it if all the tests pass. It's not like it's retuning anything we need.